src/components/common/AudioLibrarySelector.vue

```diff
@@ -68,7 +68,7 @@
         this.resource_setOrCreateValueModeSpecific({
           resourceId: this.resourceId,
           filter: {languageId: langId, contentType: SupportedContentType.AUDIO, modes: [SupportedMode.IVR]},
-          value: value.filename
+          value: value.description,
```

Comment:
@bzabos , this is the another change we should do if we sent description instead of filename. And I think it's making sense as we have now the correct file name in UI instead of having a long url path as before.

<img width="376" alt="Screen Shot 2020-12-08 at 9 55 32 AM" src="https://user-images.githubusercontent.com/68745918/101506925-94f6fb80-393b-11eb-98d6-b090dca0e26e.png">